src/tag-input/src/TagInput.js

```diff
@@ -166,30 +176,118 @@ const TagInput = memo(
+          <Autocomplete
+            onChange={changedItem => {
+              addTags(changedItem)
+              setInputValue('')
+            }}
+            items={hasAutocomplete ? autocompleteItems : []}
+            id={inputId}
+            selectedItem=""
+            inputValue={inputValue}
+          >
+            {autocompleteProps => {
+              const {
+                closeMenu,
+                getInputProps,
+                getRef: autocompleteGetRef,
+                getToggleButtonProps,
+                highlightedIndex
+              } = autocompleteProps
+
+              const {
+                onBlur: autocompleteOnBlur,
+                onChange: autocompleteOnChange,
+                onKeyDown: autocompleteKeyDown,
+                ...autocompleteRestProps
+              } = getInputProps()
+
+              return (
+                <>
+                  <TextInput
+                    appearance="none"
+                    disabled={disabled}
+                    height={height - 4}
+                    width="auto"
+                    type="text"
+                    {...inputProps}
+                    {...autocompleteRestProps}
+                    value={inputValue}
+                    id={inputId}
+                    ref={textInputRef => {
+                      autocompleteGetRef(textInputRef)
+                      if (inputRef instanceof Function) {
+                        inputRef(textInputRef)
+                      } else if (inputRef) {
+                        inputRef.current = textInputRef
+                      }
+                    }}
+                    onBlur={e => {
+                      autocompleteOnBlur(e)
+                      if (inputProps.onBlur) {
+                        safeInvoke(inputProps.onBlur, e)
+                      }
+                    }}
+                    onFocus={e => {
+                      handleInputFocus(e)
+                      if (inputProps.onFocus) {
+                        safeInvoke(inputProps.onFocus, e)
+                      }
+                    }}
+                    onChange={e => {
+                      handleInputChange(e)
+                      autocompleteOnChange(e)
+                    }}
+                    onKeyDown={e => {
+                      autocompleteKeyDown(e)
+                      if (e.key === 'Backspace' || !(highlightedIndex > -1)) {
+                        handleKeyDown(e)
+                        if (e.key === GET_KEY_FOR_TAG_DELIMITER[tagSubmitKey]) {
+                          closeMenu()
+                          setInputValue('')
+                        }
+                      }
+                      if (e.key === 'Backspace' && e.target.selectionEnd === 0) {
+                        closeMenu()
+                      }
+                    }}
+                  />
+                  {hasAutocomplete && (
+                    <Pane
+                      as="button"
```

Comment:
Can this be our `<Button />` component instead? Or does it have additional styling/functionality that's not desirable here?

Reply:
Good question, I'll see if I can use a `Button` with `appearance="none"` instead.